RadarSDK/RadarOfflineManager.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  File.swift
+//  RadarSDK
+//
+//  Created by Kenny Hu on 10/16/24.
+//  Copyright © 2024 Radar Labs, Inc. All rights reserved.
+//
+
+import Foundation
+import CoreLocation
+
+@objc(RadarOfflineManager) class RadarOfflineManager: NSObject {
+    @objc public static func contextualizeLocation(_ location: CLLocation, completionHandler: @escaping (RadarConfig?) -> Void) {
```

Comment:
I intentionally kept it more vague, it was my understanding that this might become a place where stuff like client side event generation and trip updates may occur?

Reply:
`updateTrackingOptionsFromOfflineLocation`